### ISSUE TITLE
IMDB id

### ISSUE THREAD

**Issue by petrsimek** [Mikulas/csfd-api#13]
Could you please add parsing IMDB id (or at least IMDB url) of the movie for /movie/[i:id] API call?
Thank you.

**Comment by Mikulas** [comment]
It's in 2.0 roadmap. Web version will be hopefully switched to 2.0 in roughly two months.

**Comment by petrsimek** [comment]
Thank you for the answer!

**Comment by PeterAndreus** [comment]
Is this already implemented?

**Comment by Mikulas** [comment]
Nope, sorry. As there are legal issue with this project and I'm not too eager to publish the new version. Feel free to send a PR against the 1.x branch though, that I would merge and publish easily.
